docs/conf.py

```diff
@@ -157,14 +157,16 @@ class CqlLexer(BashLexer):
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master']
+BRANCHES = ['branch-4.3', 'branch-4.4', 'branch-4.5']
```

Comment:
```suggestion
BRANCHES = ['branch-4.4', 'branch-4.5']
```

Reply:
- Branch 4.3 does not have sphinx support.
- The project can build also for ``master`` branch``